apis/v1alpha1/workspace_types.go

```diff
@@ -55,6 +55,7 @@ type VarFile struct {
 
 	// Format of this vars file.
 	// +kubebuilder:default=HCL
+	// +optional
 	Format VarFileFormat `json:"format"`
```

Comment:
We should make this a pointer since it's now `+optional`, per Kube API conventions.

Reply:
@negz are you sure it is valid for `string` not `struct` 
I checked https://book.kubebuilder.io/cronjob-tutorial/api-design.html, see ConcurrencyPolicy there which is very similar to `VarFileFormat`, no pointers

Reply:
Yes - this applies to strings too. Or more specifically "any type without a built-in nil value" per https://github.com/kubernetes/community/blob/3cb570/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required. See also https://github.com/kubernetes/community/blob/3cb570/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required which has at least one `*string` in the example.

Reply:
@negz thanks, does it look better now?

Reply:
It does, thank you!